Libraries/Text/RCTTextAttributes.m

```diff
@@ -252,6 +252,23 @@ - (RCTUIColor *)effectiveBackgroundColor // TODO(OSS Candidate ISS#2710739)
   return effectiveBackgroundColor ?: [RCTUIColor clearColor]; // TODO(OSS Candidate ISS#2710739)
 }
 
+static NSString *capitalizeText(NSString *text)
```

Comment:
why is this change better than the os provided capilizeString?

Reply:
Here's the original PR: https://github.com/facebook/react-native/pull/32774

The bug still repros in Swift 5.6, so it might be worth filing an Apple bug, but that's a separate issue.